introspection-engine/connectors/sql-introspection-connector/src/rendering/defaults.rs

```diff
@@ -50,6 +50,15 @@ pub(crate) fn render<'a>(field: ScalarFieldPair<'a>, warnings: &mut Warnings) ->
             DefaultKind::Autoincrement => Some(renderer::DefaultValue::function(Function::new("autoincrement"))),
             DefaultKind::Uuid => Some(renderer::DefaultValue::function(Function::new("uuid"))),
             DefaultKind::Cuid => Some(renderer::DefaultValue::function(Function::new("cuid"))),
+            DefaultKind::Nanoid(length) => {
+                let mut fun = Function::new("nanoid");
+
+                if let Some(param) = length.filter(|i| (*i) > &2) {
```

Comment:
Why do we filter the length here. If it's invalid, we silently just do nothing? Didn't we already have a validation for this?

Reply:
That's a great point - there _is_ indeed already a separate validation for it